### DISCUSSION TITLE
Export to file

### DISCUSSION THREAD

**Discussion by DanielMadmon** [larq/larq#886] [Q&A]
How can I export the trained model and weihgts to a file

**Comment by CNugteren** [comment]
If you want to use it for inference, you can export it as a TFLite FlatBuffer, see [here](https://docs.larq.dev/compute-engine/end_to_end/#converting-the-model). Otherwise, if you want to use it for further training, you can use the normal Keras loading and saving functionality.

See also https://github.com/larq/larq/discussions/760.
